### PR TITLE
Avoid using "Node" and "Node's"

### PR DESCRIPTION
**Pull request by tniessen**
As per our branding guidelines, both should be avoided, and Node.js should be used instead.

(I think a lot of these documents should be updated / removed anyway...)

### CODE REVIEW

**Review by Trott — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
@nodejs/website: Other languages, please notice this issue ;)

**Review by 3imed-jaberi — APPROVED** [review]

File: locale/ar/docs/guides/anatomy-of-an-http-transaction.md
Comment (tniessen) [inline locale/ar/docs/guides/anatomy-of-an-http-transaction.md]: @3imed-jaberi `node.js` is as wrong as `Node`. As per our branding guidelines, `node` is allowed when referring to the executable, and `Node.js` is to be used for the product in all other contexts. Neither `Node` nor `node.js` are correct.

**Review by tniessen — COMMENTED** [review]
Apparently I cannot request changes on my own PR, but please fix your commits @3imed-jaberi.

File: locale/ar/docs/guides/anatomy-of-an-http-transaction.md
Comment (3imed-jaberi) [reply locale/ar/docs/guides/anatomy-of-an-http-transaction.md]: typo beceause i use change multiple !

**Review by 3imed-jaberi — COMMENTED** [review]
@Aissaoui-Ahmed check my ar changes !

**Review by aymen94 — APPROVED** [review]

File: locale/ar/docs/guides/anatomy-of-an-http-transaction.md
Comment (nschonni) [inline locale/ar/docs/guides/anatomy-of-an-http-transaction.md]: This is a markdown shortcut link that relys on the spelling to match at the bottom of the page

File: locale/ar/docs/guides/debugging-getting-started.md
Comment (nschonni) [inline locale/ar/docs/guides/debugging-getting-started.md]: Should be capitialized unless the original text was talking about the executable binary
```suggestion
في ذلك الحاسوب البعيد، عليك بدء عملية Node.js مع جعل المدقق يستمع على المضيف المحلي فقط (وهو الافتراضي).
```

File: locale/en/docs/guides/dont-block-the-event-loop.md
Comment (nschonni) [inline locale/en/docs/guides/dont-block-the-event-loop.md]:
```suggestion
After consulting the Node.js [File system](https://nodejs.org/api/fs.html) APIs, you opted to use `fs.readFile()` for simplicity.
```

File: locale/en/knowledge/getting-started/globals-in-node-js.md
Comment (nschonni) [inline locale/en/knowledge/getting-started/globals-in-node-js.md]:
```suggestion
* `process` - The Node.js built-in `process` module, which provides interaction with the current Node.js process. [Read More](/en/knowledge/getting-started/the-process-module/)
```

File: locale/en/knowledge/getting-started/globals-in-node-js.md
Comment (nschonni) [inline locale/en/knowledge/getting-started/globals-in-node-js.md]:
```suggestion
* `console` - The Node.js built-in `console` module, which wraps various STDIO functionality in a browser-like way. [Read More](/en/knowledge/getting-started/the-console-module/)
```

**Review by nschonni — CHANGES_REQUESTED** [review]
I think the i18n changes should be submitted separately, since they are making an already large PR bigger

**Comment by nschonni** [comment]
Not sure if there is more to clean up after this or if the Remark rule can be turned on after this https://github.com/nodejs/nodejs.org/blob/ce6c5dd5cd64027fc016cb80ad7418c29ce7af3d/.remarkrc#L14

File: locale/ar/docs/guides/debugging-getting-started.md
Comment (tniessen) [reply locale/ar/docs/guides/debugging-getting-started.md]: https://github.com/nodejs/nodejs.org/pull/2914#discussion_r371007977

File: locale/ar/docs/guides/anatomy-of-an-http-transaction.md
Comment (Aissaoui-Ahmed) [reply locale/ar/docs/guides/anatomy-of-an-http-transaction.md]: if you change it, the link will not be able to redirect!

**Review by Aissaoui-Ahmed — APPROVED** [review]

File: locale/ar/docs/guides/anatomy-of-an-http-transaction.md
Comment (nschonni) [reply locale/ar/docs/guides/anatomy-of-an-http-transaction.md]: An alternate fix would be to keep the refname in the second bracket like `[هنا][API reference]`

**Review by nschonni — CHANGES_REQUESTED** [review]

**Comment by tniessen** [comment]
Ping @ whoever needs to fix stuff.

File: locale/ar/docs/guides/anatomy-of-an-http-transaction.md
Comment (nschonni) [inline locale/ar/docs/guides/anatomy-of-an-http-transaction.md]: This still looks like it's breaking the Markdown format since the square brackets are now separated.

File: locale/ar/docs/guides/anatomy-of-an-http-transaction.md
Comment (nschonni) [inline locale/ar/docs/guides/anatomy-of-an-http-transaction.md]: This also breaks the anchor link above. You need to make the same change, or leave this change, and translate the right square brackets, while copying the english into the left square brackets of the reference

**Review by nschonni — CHANGES_REQUESTED** [review]

File: locale/ar/docs/guides/anatomy-of-an-http-transaction.md
Comment (nschonni) [reply locale/ar/docs/guides/anatomy-of-an-http-transaction.md]: The GitHub preview seems happy, but I'd double check that the markdown is library looks OK

**Comment by tniessen** [comment]
I removed the merge commit, rebased properly, and cleaned up the commit history. Another ping @ whoever needs to fix stuff.

To be honest, I am not super happy about other maintainers pushing commits to my PR if those block an originally uncontroversial PR for weeks. Sure, I can disable editing by maintainers for this repo from now on, but this behavior might still be frustrating for others, especially new contributors.

**Comment by XhmikosR** [comment]
@tniessen how about you rebase and drop any other patches so that we can land this and translators continue in separate PRs?

File: locale/en/knowledge/command-line/how-to-prompt-for-command-line-input.md
Comment (XhmikosR) [inline locale/en/knowledge/command-line/how-to-prompt-for-command-line-input.md]: `node's` should be changed here maybe?

File: locale/en/knowledge/cryptography/how-to-use-crypto-module.md
Comment (XhmikosR) [inline locale/en/knowledge/cryptography/how-to-use-crypto-module.md]: Node should be Node.js?

**Comment by tniessen** [comment]
> how about you rebase and drop any other patches so that we can land this and translators continue in separate PRs?

@XhmikosR I have rebased and dropped the offending commits. @nschonni PTAL.

In my opinion, it should be unacceptable for maintainers to block a PR **for an entire month** by pushing their own changes to the author's branch without the author's approval, regardless of whether the author is a first-time contributor or a long-term project member. I have learned my lesson, I will disable editing for collaborators for this repository from now on, but if this is recurring behavior from members of the website group, I suggest to review the group's members and privileges.

File: locale/en/knowledge/cryptography/how-to-use-crypto-module.md
Comment (tniessen) [inline locale/en/knowledge/cryptography/how-to-use-crypto-module.md]:
```suggestion
The crypto module is mostly useful as a tool for implementing [cryptographic protocols](https://en.wikipedia.org/wiki/Cryptographic_protocol) such as [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) and [https](https://en.wikipedia.org/wiki/Https). For most users, the built-in [tls module](https://nodejs.org/api/tls.html) and [https module](https://nodejs.org/api/https.html) should more than suffice. However, for the user that only wants to use small parts of what's needed for full-scale cryptography or is crazy/desperate enough to implement a protocol using OpenSSL and Node.js: Read on.
```

File: locale/en/knowledge/command-line/how-to-prompt-for-command-line-input.md
Comment (tniessen) [inline locale/en/knowledge/command-line/how-to-prompt-for-command-line-input.md]:
```suggestion
Streams are the Node.js way of dealing with evented I/O - it's a big topic, and you can read more about them [here](https://nodejs.org/api/stream.html). For now, we're going to use the built-in `readline` module which is a wrapper around Standard I/O, suitable for taking user input from command line(terminal).
```

**Comment by XhmikosR** [comment]
I agree with your statement, this was a simple change and should have stayed like that. I don't think there will be an issue in the future, but it's definitely something to keep in mind @Trott.

Now, back to this, I only have 2 comments. There's probably are more instances but we can take another stab at it later.

File: locale/en/docs/guides/debugging-getting-started.md
Comment (XhmikosR) [inline locale/en/docs/guides/debugging-getting-started.md]:
```suggestion
`--inspect` and Inspector instead.**
```

File: locale/en/knowledge/getting-started/what-is-require.md
Comment (XhmikosR) [inline locale/en/knowledge/getting-started/what-is-require.md]:
```suggestion
The rules of where `require` finds the files can be a little complex, but a simple rule of thumb is that if the file doesn't start with "./" or "/", then it is either considered a core module (and the local Node.js path is checked), or a dependency in the local `node_modules` folder. If the file starts with "./" it is considered a relative file to the file that called `require`. If the file starts with "/", it is considered an absolute path. NOTE: you can omit ".js" and `require` will automatically append it if needed. For more detailed information, see [the official docs](https://nodejs.org/docs/v0.4.2/api/modules.html#all_Together...)
```

Comment (XhmikosR) [reply locale/en/knowledge/getting-started/what-is-require.md]: ^^404 link I noticed by accident :)

**Review by nschonni — APPROVED** [review]

**Pull request merged**
